# 0 (close) 1 (open)
int32 mode
# 1500 mv to 4500 mv, if 0 set to 1500
int32 voltage

